src/content/en/showcase/2016/service-worker-perf.md

```diff
@@ -0,0 +1,679 @@
+project_path: /web/_project.yaml
+book_path: /web/showcase/_book.yaml
+description: Measuring the Real-world Performance Impact of Service Workers
+
+{# wf_published_on: 2016-07-22T18:00:00.000Z #}
+{# wf_updated_on: 2016-07-22T18:00:00.000Z #}
+{# wf_author: philipwalton #}
+{# wf_featured_image: /web/showcase/2016/images/service-worker-perf/desktop-all.png #}
+{# wf_featured_snippet: One of the most significant benefits of service workers (from a performance perspective, at least) is their ability to proactively control the caching of assets. A web application that can cache all of its necessary resources should load substantially faster for returning visitors. But what do these gains actually look like to real users? And how do you even go about measuring this? #}
+
+<style>
+td[align="right"] {
+  text-align: right!important;
+}
+.pull-quote {
+  font-size: 1.75em;
+  font-style: italic;
+  font-weight: 300;
+  line-height: 1.2;
+  padding: 1em 2em;
+}
+</style>
+
+# Measuring the Real-world Performance Impact of Service Workers {: .page-title }
+
+One of the most significant benefits of [service workers](/web/fundamentals/primers/service-worker/) (from a performance perspective, at least) is their ability to proactively control the caching of assets. A web application that can cache all of its necessary resources should load substantially faster for returning visitors. But what do these gains actually look like to real users? And how do you even go about measuring this?
+
```

Comment:
And how do you even go about measuring this? ca be shortened to 'And how do you even measure this?'